tox.ini

```diff
@@ -61,6 +65,7 @@ deps =
     sphinx: aiohttp
     sphinx: git+https://github.com/tornadoweb/tornado.git
 
+    py3-pymongo-v3.12: git+https://github.com/mongodb/mongo-python-driver.git@v3.12
```

Comment:
You can remove this - we now install pymongo for this test in the `commands` section.